src/main.rs

```diff
@@ -19,7 +19,7 @@ async fn main() -> anyhow::Result<()> {
     init_tracing();
 
     let args = Args::parse();
-    let cfg: Config = Config::from_file(args.config_file)?;
+    let cfg: Config = Config::from_file_and_env(args.config_file)?;
```

Comment:
This method name does not imply that you have to supply both a file and environment variables.  The tests below demonstrate that